glean/src/server_events.rs

```diff
@@ -0,0 +1,308 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Required imports
+use chrono::Utc;
+use serde::{Deserialize, Serialize};
+use serde_json;
```

Comment:
Normally you don't have to specify crate roots like this. You only need to specify the various elements you're importing. Rust will automatically associate crates when you specify the root (like you do later for `Value`.)

If you want, you could specify
```suggestion
use serde_json::{Value, to_string};
```
and then drop the crate prefix from those uses later.

(I'm surprised that clippy didn't yell about this.)